Xamarin.Essentials/Types/PlatformExtensions/ColorExtensions.shared.cs

```diff
@@ -0,0 +1,47 @@
+﻿#if !NETSTANDARD1_0
+using System;
+using System.Drawing;
+
+namespace Xamarin.Essentials
+{
+    public static partial class ColorExtensions
```

Comment:
Maybe we can add another `WithAlpha` - I find myself want a color, but with a tweaked alpha because of styling. This is what Material is based on in fact, a color with varying degrees of alpha.

Reply:
Implemented!